src/tidy.zig

```diff
@@ -74,7 +74,9 @@ test "tidy changelog" {
             return error.TrailingWhitespace;
         }
         const line_length = try std.unicode.utf8CountCodepoints(line);
-        if (line_length > 100) {
+        const has_link = std.mem.indexOf(u8, line, "http://") orelse
+            std.mem.indexOf(u8, line, "https://");
+        if (line_length > 100 and has_link == null) {
```

Comment:
:+1: I was racking my brain a bit last week how to elegantly parse URLs without too much complexity, and it didn't occur to me to just look for `https://`!